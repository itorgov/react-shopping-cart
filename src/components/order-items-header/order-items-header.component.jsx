import React from 'react';

import styles from './order-items-header.module.scss';

export const OrderItemsHeader = () => (
    <div className={styles.header}>
        <div>Product</div>
        <div>Price</div>
        <div>Qty</div>
        <div>Total</div>
        <div>&nbsp;</div>
    </div>
);
