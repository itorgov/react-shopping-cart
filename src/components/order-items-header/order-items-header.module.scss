.header {
  padding: 0;
  text-transform: uppercase;
  letter-spacing: 1px;
  font-size: 1.2rem;
  font-weight: 600;
  color: var(--color-text-light);

  display: flex;
  align-items: center;

  & div {
    flex: 1;
    display: flex;
    align-items: center;
  }

  & div:first-child {
    flex: 2;
  }

  & div:last-child {
    flex: 0;
    min-width: 2rem;
  }
}
