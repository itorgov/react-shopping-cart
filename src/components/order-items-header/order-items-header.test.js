import React from "react";
import {shallow} from "enzyme";

import {OrderItemsHeader} from './order-items-header.component';

describe('<OrderItemsHeader />', () => {
    it('expect to render', () => {
        expect(shallow(<OrderItemsHeader />)).toMatchSnapshot();
    });
});
