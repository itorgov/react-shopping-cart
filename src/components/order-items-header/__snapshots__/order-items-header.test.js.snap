// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<OrderItemsHeader /> expect to render 1`] = `
<div
  className="header"
>
  <div>
    Product
  </div>
  <div>
    Price
  </div>
  <div>
    Qty
  </div>
  <div>
    Total
  </div>
  <div>
    Â 
  </div>
</div>
`;
